src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -48,7 +48,13 @@ to the application.
 
 ## Metrics
 
-### GC related metrics
+> [!NOTE]
+> .NET 9 introduced built-in runtime metrics. As such, when applications target
+  .NET 9 or greater this package registers a `Meter` to receive the built-in
+  `System.Runtime` metrics. See the [.NET Runtime metrics documentation](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/built-in-metrics-runtime)
+  for details of the metric and attribute names for the built-in metrics.
+
+### GC related metrics for .NET Framework
```

Comment:
```suggestion
### GC related metrics
```

Why was `for .NET Framework` added? None of the metrics is unique to .NET Framework.